fi.json

```diff
@@ -239,113 +239,113 @@
+			"option-config-location-description": "Käytä muuta määrityskansiota kuin oletuskansiota. Sen täytyy alkaa pisteellä.",
+			"option-config-location-warning": "Määrityksien sijainti tulee olla kelvollinen kansion nimi, joka alkaa pisteellä.",
 			"label-license": "Lisenssi",
-			"label-your-commercial-license-key": "Sinun kaupallinen lisenssiavaimesi on \"{{key}}\".",
-			"label-commercial-license-info": "Rekisteröity yritykselle \"{{company}}\", jossa on {{seats}} käyttäjää. Kelvollinen {{expiry}} asti.",
-			"label-validating-commercial-license": "Vahvistetaan...",
+			"label-your-commercial-license-key": "Kaupallinen lisenssiavaimesi on \"{{key}}\".",
+			"label-commercial-license-info": "Rekisteröity yritykselle \"{{company}}\", jossa on {{seats}} käyttäjää. Lisenssi on voimassa {{expiry}} asti.",
+			"label-validating-commercial-license": "Tarkistetaan...",
 			"label-invalid-commercial-license": "Lisenssi on virheellinen: ",
 			"button-remove-commercial-license": "Poista",
 			"option-catalyst": "Catalyst-lisenssi",
 			"option-catalyst-desc": "Sinulla on tällä hetkellä {{tier}} Catalyst-lisenssi. Kiitos tuestasi!",
-			"option-catalyst-desc-no-license": "Sinulla ei tällä hetkellä ole Catalyst-lisenssiä. Catalyst-lisenssi on kertalahjoitus Obsidianin kehityksen tukemiseen.",
+			"option-catalyst-desc-no-license": "Sinulla ei tällä hetkellä ole Catalyst-lisenssiä. Catalyst-lisenssi on kertalahjoitus, jonka avulla Obsidian voi olla täysin käyttäjiensä tukema. Lisenssi oikeuttaa pääsyn uusiin toiminnallisuuksiin ensimmäisten joukossa.",
 			"label-app": "Sovellus",
-			"label-add-own-language": "Katso kuinka voit lisätä oman kielesi Obsidian-sovellukseen"
+			"label-add-own-language": "Lue, kuinka voit kääntää Obsidianin omalle kielellesi."
 		},
 		"account": {
-			"name": "Käyttäjä",
-			"option-your-account": "Sinun käyttäjäsi",
-			"option-your-account-desc": "Olet tällä hetkellä kirjautunut sisään {{name}} ({{email}}).",
-			"option-your-account-desc-no-login": "Et olet kirjautunut sisään tällä hetkellä. Käyttäjää tarvitaan vain Obsidian Synciin, Obsidian Publishiin, ja Catalyst insider-versioihin.",
-			"label-insider": "insider",
-			"label-supporter": "supporter",
+			"name": "Tili",
+			"option-your-account": "Käyttäjätilisi",
+			"option-your-account-desc": "Olet kirjautunut tilille {{name}} ({{email}}).",
+			"option-your-account-desc-no-login": "Et ole kirjautunut sisään. Käyttäjätili vaaditaan vain Obsidian Synciin, Obsidian Publishiin ja kokeiluversioihin.",
+			"label-insider": "Testaaja",
+			"label-supporter": "Tukija",
 			"label-vip": "VIP",
 			"button-upgrade-catalyst": "Päivitä",
 			"option-commercial-license": "Kaupallinen lisenssi",
-			"option-commercial-license-desc": "Jos käytät Obsidian-sovellusta kaupallisiin tarkoituksiin, saatat tarvita kaupallisen lisenssin.",
+			"option-commercial-license-desc": "Auta Obsidiania pysymään täysin käyttäjiensä rahoittamana.",
```

Comment:
Please don't translate the word "Obsidian"

Reply:
Obsidian is a brand name that should not be changed

Reply:
I understand that, but inflection is rather integral to the grammar here. I can change it though to shift the inflection to another word. It'll make the affected strings somewhat longer and more stilted, but will still be perfectly understandable.